### DISCUSSION TITLE
Page Refresh after deleting item

### DISCUSSION THREAD

**Discussion by xrgarcia** [bigskysoftware/htmx#1717] [General]
Howdy! I've been using htmx for a few weeks now and i've run into a snag. I'm sure it's my lack of knowledge. I want to trigger a page refresh after http-delete is triggered so the list of records gets updated.

My button is configured as such
<button type="button" class="btn btn-danger" hx-delete="/pages/watchlist.html?ticker={{company.ticker}}" >Delete</button></td>

When i click this, the record is deleted, and i can see this by refreshing the page.  Can someone point me at a doc or give advice?

Warm Regards,
-ray

**Comment by Telroshan** [comment]
Hey, you may send the [HX-Refresh](https://htmx.org/reference/#response_headers) header to trigger page refresh, hope that helps!

**Comment by xrgarcia** [comment]
YAAAS!! that works. Thank you! very much digging this framework. thanks for the good work.
